### DISCUSSION TITLE
MyHyundai update starting August 19th in Canada

### DISCUSSION THREAD

**Discussion by zadigre** [Hyundai-Kia-Connect/kia_uvo#349] [General]
Hello Everyone, 

yesterday I received an email from Hyundai Canada saying starting August 19th MyHyundai with Bluelink app will replace the current Bluelink app.

does anyone know if this change will be impacting the integration? If possible, should be wait before updating the app just in case this break something with our Bluelink account and this integration?

**Comment by cdnninja** [comment]
This sounds much like the rebrand on the Kia side. It had no impact. 

If it does we can address it at that time.

**Reply by zadigre** [reply]
good to know...
I guess this is just a frontend change of the app... but it will still make the same calls on the backend.
so lets see tomorrow how it goes... I'll report back to confirm if all is fine.

**Comment by Kevr0n228** [comment]
I updated to the new version of the app this morning and all day I've been having a lot of trouble with bluelink. It's even slower than usual and I wasn't able to use the HA integration most of the morning (though that seems to be on hyundai's side).

**Reply by zadigre** [reply]
I just read on some other Facebook groups that Hyundai is having problem with the Bluelink data/server at this moment... they are rolling back data to the previous state. Tomorrow should apparently be better.
